src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -299,6 +279,8 @@ export default class RadarTargetModel {
     */
     enable() {
         this._eventBus.on(EVENT.SET_THEME, this._setTheme);
+
+        return this;
```

Comment:
👍 just add `@chainable` to the docblock to whenever we return `this`.

Reply:
If enable() comes at the end, it shouldn't need to be chainable, right?

Reply:
all the stuff where we initialize is fine to be chainable (so .init(), .reset(), .enable(), .disable(), ._initSomethingInParticular(), etc). Normal methods like .doSomething() should not be chainable.